Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -1534,6 +1534,20 @@ - (void)showReadingThemesControlsPopup {
     [self.readingThemesControlsPresenter objCShowReadingThemesControlsPopupOn:self theme:self.theme];
 }
 
+#pragma mark - Scrolling to anchors
+
+- (void)scrollToAnchor:(nullable NSString *)anchor animated:(BOOL)animated completion:(nullable dispatch_block_t)completion {
+    self.updateTableOfContentsSectionOnScrollEnabled = NO;
+    [self.webViewController scrollToAnchor:anchor
```

Comment:
@joewalsh It seems the problem is related to this... if the `scrollToAnchor` call here is commented out the exception goes away but the article doesn't load at all (when tapping the `geography section` link on `testwiki > bird`). So I guess it makes sense we're seeing the exception since it's trying to scroll when the article isn't loaded?

Reply:
Opps it's completion block calls `showWebView` so that's why it wasn't loading... hmm...

Reply:
f.t.r i was typing too fast above - i meant the cause of the assertion not `exception` 🤦‍♂️